src/import-data/ImportData.test.ts

```diff
@@ -261,18 +279,17 @@ describe('ImportData', () => {
 
   describe('TDR', () => {
     describe('snapshot exports', () => {
-      const queryParams = {
-        format: 'tdrexport',
-        snapshotId: '00001111-2222-3333-aaaa-bbbbccccdddd',
-        tdrmanifest: 'https://example.com/path/to/manifest.json',
-        tdrSyncPermissions: 'true',
-        url: 'https://data.terra.bio',
-      };
-
       it('imports snapshot exports into Google workspaces', async () => {
         // Arrange
         const user = userEvent.setup();
 
+        const queryParams = {
```

Comment:
Seems like the only interesting variation of these queryParams for these two tests is with the `snapshotId`; maybe factor out the other params into a generically named const like `tdrExportParams` (or a better that more clearly describes the unique qualities of the set of defaults), and let the test use the spread operator to _only_ provide the `snapshotId` to more clearly tie what we're fiddling with in the fixture to what's being tested, eg:

`{ ...tdrExportParams, snapshotId: googleSnapshotFixture.id }`